.github/workflows/nix_linux_x86_64.yml

```diff
@@ -24,7 +24,12 @@ jobs:
         run: nix develop -c ./ci/roc_test_builtins.sh
 
       - name: test wasm32 cli_tests
-        run: nix develop -c cargo test --locked --release --features="wasm32-cli-run"
+        # skipping glue tests due to difficult multithreading bug, we run them single threaded in the next step
+        run: nix develop -c cargo test --locked --release --features="wasm32-cli-run" -- --skip glue_cli_tests
+
+      - name: wasm32 glue_cli_tests
+        # single threaded due to difficult bug when multithreading
+        run: nix develop -c cargo test --locked --release --features="wasm32-cli-run" glue_cli_tests -- --test-threads=1
```

Comment:
This feels like it shouldn't be part of this PR. Nothing touches wasm.

Reply:
Ah, I see that @Anton-4  added this as a workaround of some sort.

Reply:
I see you merged, but just for posterity: we are all confused why this PR seems to trigger some strange behaviour with this test. @Anton-4 added this to help CI accept the PR.

Reply:
Yeah, not caused by this PR but I just included the fix here